github/ci/prow-deploy/files/jobs/kubevirt/terraform-provider-kubevirt/terraform-provider-kubevirt-presubmits.yaml

```diff
@@ -0,0 +1,76 @@
+presubmits:
+  kubevirt/terraform-provider-kubevirt:
+  - name: pull-terraform-proider-kubevirt-e2e-k8s-1.20
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
+    always_run: true
+    optional: false
+    skip_report: false
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      timeout: 7h
```

Comment:
I figure you'll change this after a few runs when you know the average run time on the cluster?

Reply:
I changed it to 1h, I will change it later if needed